hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/request/EntityRequest.java

```diff
@@ -2,11 +2,12 @@
 
 import java.util.List;
 import java.util.Optional;
+import org.hypertrace.core.graphql.common.request.ContextualRequest;
 import org.hypertrace.core.graphql.common.request.ResultSetRequest;
 import org.hypertrace.graphql.metric.request.MetricRequest;
 import org.hypertrace.graphql.metric.schema.argument.AggregatableOrderArgument;
 
-public interface EntityRequest {
+public interface EntityRequest extends ContextualRequest {
```

Comment:
nit - the resultSetRequest this contains already has a context on it - rather than adding it in the various impls, you can probably just add
```java
  default GraphQlRequestContext context() {
    return resultSetRequest().context();
  }
```